### DISCUSSION TITLE
Q: Monitoring by an external tool

### DISCUSSION THREAD

**Discussion by zbalkan** [semaphoreui/semaphore#1760] [Q&A]
Hi all,

If we have a monitoring tool like Zabbix or Grafana, and want to let Semaphore to send notifications on task start, completion and failures, what's the recommended way?

**Comment by tboerger** [comment]
There is an open pull request which introduces generic webhook support, with that it should be possible to track any changes within an external tool.

Beside that there is still an open feature request to implement a prometheus exporter, which could also be a used for monitoring.
